fem/bilininteg.hpp

```diff
@@ -2995,6 +2997,26 @@ class ElasticityIntegrator : public BilinearFormIntegrator
    Vector divshape;
 #endif
 
+   // PA extension
+   std::shared_ptr<QuadratureFunction> lambda_quad, mu_quad;
+   std::shared_ptr<QuadratureFunction> q_vec;
+   std::shared_ptr<QuadratureSpace> quad_space;
```

Comment:
Are these `shared_ptr` because they are shared by the children "component integrators"? The way this is designed is somewhat confusing.